Sources/Rendering/Core/RenderWindowInteractor/index.d.ts

```diff
@@ -113,398 +127,398 @@ export interface vtkRenderWindowInteractor extends vtkObject {
 	/**
 	 *
 	 */
-	invokeStartAnimation(callData: object): any;
+	invokeStartAnimation(callData: IRenderWindowInteractorEvent): any;
```

Comment:
The [invoke${_capitalize(eventName)}](https://github.com/Kitware/vtk-js/blob/master/Sources/macros.js#L941)  calls `invoke` which returns void i guess, can we have void on them instead of any?

Reply:
I confirm, `invoke` return `void`.

Reply:
Thanks Sébastien